All EIPIP Meetings/Meeting 012.md

```diff
@@ -0,0 +1,364 @@
+
+(James) - As long as we're not listing out more people in teh informational EIP, like if you document the process, and maybe shephard a few people whom want ot know more about it.
+
+(William) - In other words the EIP should be process; a contact list shouldn't be part of an EIP, especially because that's not something to be set in stone and would mutate with time. So what we would be looking for in an EIP would be some sort of codefication of process but not the specific contacts. It would probably suffice to say major stakeholder or something vague like that instead of providiing contact information
+
+(Pooja) - It would make sense to create a checklist kind of thing at the time said situation emerges. And maybe adding some way to contact people whom are actively involved with the script.
+
+(William) - So it sounds like this should probably be pulled off of the EIPIP agenda and then maybe at such a time when we're ready to submit an EIP it would maybe come back again as an item. Does that sound about right?
+
+(Pooja) - Sounds good to me
+
+(James) - I'd also ask some editors about what they think; I'd recommend that, and everything else sounds good.
+
+## 5.1. Decisions
+- *12.5* - Remove emergency EIP process item until such time that it becomes pertinent
+
+# 6. Various Final Topics
+
+Video | [56:08](https://youtu.be/5PSQDT1nr2E)
+
+## 6.1. EIPIP project board
+ 
+(Pooja) - Okay, so we are almost at time but I want to share a couple of things here. We have created this EIP project code, the link is available in the agenda. Everyone can go ahead and look adn check that out. We'd love to have feedback.
+
+(James) - It's so good!
+
+(Pooja) - Thank you ^_^. so we would like to keep it as up to date with the tasks that we are completing, we could then let people go back and see all that we have achieved with this group. Which, would be a great informational sharing platform that would encourage others to be a part of this community whom care about developing Ethereum. There's one last thing that I would like to quickly share about one thing. In the previous call we shared the upgrade post-mortem. But the thing that we did not share was the general template. I'm sharing the link in the chat if people can give it a look, and if they have any feedback or comment, feel free to let us know. Traditionally we have more time to discuss anything else. Anyone else who'd like to discuss something, please speak freely.
+
+...
+
+(James) - Looks good to me, but I'm biased
+
+(Pooja) - (laughing) yeah, we worked together on that; so yeah, we would like to hear more feedback from people and maybe again we will stop at this point and have the cat hearders have some kind of discussion about this kind of EIP. I'm planning to start an ETH magician post about it.
+
+## 6.2. Clarifications for how a precompile address is decided
+
+(Edson) - It might be good to open a PR and then change the EIP number

```

Comment:
This is a suggestion for the Upgrade postmortem template and not for precompile address.